src/v3/src/util/mergeThemes.ts

```diff
@@ -73,6 +73,12 @@ export const mergeThemes = (
                   // @ts-expect-error FIXME CSSInterpolation may not be CSSObject
                   ...resolve(config.styleOverrides?.icon, options),
                 }),
+                input: (options: Record<string, unknown>) => ({
+                  // @ts-expect-error FIXME CSSInterpolation may not be CSSObject
+                  ...resolve(prev.components?.[component]?.styleOverrides?.input, options),
+                  // @ts-expect-error FIXME CSSInterpolation may not be CSSObject
+                  ...resolve(config.styleOverrides?.input, options),
+                }),
```

Comment:
what does this do?

Reply:
This utility merges the MUI theme configuration -- right now it's not fully generalized for the properties within `styleOverrides` so I'm just duplicating the inner logic for each one we need to handle. @lesterchoi-okta is updating the utility to eventually handle this better but for expedience we're just doing this right now.